internal/core/src/query/SearchBruteForce.cpp

```diff
@@ -54,6 +56,34 @@ BruteForceSearch(const dataset::SearchDataset& dataset,
         auto base_dataset =
             knowhere::GenDataSet(chunk_rows, dim, chunk_data_raw);
         auto query_dataset = knowhere::GenDataSet(nq, dim, dataset.query_data);
+
+        auto xb = base_dataset->GetTensor();
+        auto float_xb =
+            new float[base_dataset->GetRows() * base_dataset->GetDim()];
+
+        auto xq = query_dataset->GetTensor();
+        auto float_xq =
+            new float[query_dataset->GetRows() * query_dataset->GetDim()];
+
+        if (data_type == DataType::VECTOR_FLOAT16) {
```

Comment:
add some comment here, it is a temporary solution, need to be refactor in the future.

Reply:
Get